### PR TITLE
Added questions and rules page

### PR DESCRIPTION
**Pull request by Amandeep-sinha**
please review my PR for rules an question

image
image
:
: 
image

### CODE REVIEW

**Review by siddhigate — APPROVED** [review]
Code is well structured. Good to merge👍

File: index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            </div>
+            Developed and Designed by : Amandeep<br><br>
+        </footer>
+
+</body>
+
+</html>
+
+<!--  Trivia logo:- https://bit.ly/3JHkeZs 
+        Trivia bg img :- https://bit.ly/3I6sAtl  -->
```
Comment (codewithrishu) [inline index.html:97]: remove unwanted code

File: style.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    margin: 0;
+    background-color: var(--backcolor);
+    color: var(--primary);
+}
+
+.trivia-header-div {
+    background-image: url('https://bit.ly/3p1IZrt');
+    height: 600px;
+    width: 100%;
+    padding: 0%;
```
Comment (codewithrishu) [inline style.css:23]: unit with 0 is not needed

File: ques1-1.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <div class="question-div">
+        <h2>1. What is the title of the Italian protest song which is played throughout the series?</h2>
+        <button class="block-btn">Bella ciao</button>
+        <button class="block-btn">titanium</button>
+        <button class="block-btn">senorita</button>
+        <button class="block-btn">believer</button>
+    </div>
+
+    <div class="next-div">
+        <a href="ques1-2.html"><button class="btn">Next-->></button></a>
```
Comment (codewithrishu) [inline ques1-1.html:23]: its not necessary to do but instead of writing arrow manually in next button, try using icon

**Review by codewithrishu — APPROVED** [review]
other then the changes suggested, looks good to merge

File: index.html (hunk 1 above)
Comment (Amandeep-sinha) [reply index.html:97]: okay

**Pull request merged**